app/file.go

```diff
@@ -360,7 +360,7 @@ func MigrateFilenamesToFileInfos(post *model.Post) []*model.FileInfo {
 
 func GeneratePublicLink(siteURL string, info *model.FileInfo) string {
 	hash := GeneratePublicLinkHash(info.Id, *utils.Cfg.FileSettings.PublicLinkSalt)
-	return fmt.Sprintf("%s%s/public/files/%v/get?h=%s", siteURL, model.API_URL_SUFFIX_V3, info.Id, hash)
+	return fmt.Sprintf("%s/files/%v/public?h=%s", siteURL, info.Id, hash)
```

Comment:
@jwilander please ignore this for a while. i'll get back to this later once i returned to my dev pc. thanks